tdrs-backend/docker-compose.ci.yml

```diff
@@ -0,0 +1,18 @@
+# Docker compose file intended to be used in CI/CD pipelines
+version: "3"
+
+services:
+  web:
+    environment:
+      # Additional values passed in by the CircleCI host machine
+      - ACR_VALUES=${ACR_VALUES}
+      - BASE_URL=${BASE_URL}
+      - CLIENT_ASSERTION_TYPE=${CLIENT_ASSERTION_TYPE}
+      - CLIENT_ID=${CLIENT_ID}
+      - FRONTEND_BASE_URL=${FRONTEND_BASE_URL}
+      - MOCK_TOKEN=${MOCK_TOKEN}
+      - OIDC_OP_AUTHORIZATION_ENDPOINT=${OIDC_OP_AUTHORIZATION_ENDPOINT}
+      - OIDC_OP_ISSUER=${OIDC_OP_ISSUER}
+      - OIDC_OP_JWKS_ENDPOINT=${OIDC_OP_JWKS_ENDPOINT}
+      - OIDC_OP_LOGOUT_ENDPOINT=${OIDC_OP_LOGOUT_ENDPOINT}
+      - OIDC_OP_TOKEN_ENDPOINT=${OIDC_OP_TOKEN_ENDPOINT}
```

Comment:
This may be something for another ticket, but we could negate the need for this file all together if we supply defaults for all of these values using one of the following two methods:
* (Easy option) In `wsgi.py` use `os.environ.setdefault` to specify the default values for all of these listed variables
* (Better option, more work) Utilize the Django settings modules to call `os.environ.get(x, 'DEFAULT_VAL)` for each of these variables and expose them as Django settings. From there we can just access `settings.X` instead of calling `os.environ` in multiple locations throughout the backend. This would give us one way to access these values in API code and prevent some possible unhandled exceptions that we have now if these values aren't defined.

Reply:
I like the latter solution best, and we will create a ticket to address it